app/views/admin/councillor_contributions/_details.html.arb

```diff
@@ -0,0 +1,13 @@
+table do
+  tr do
+    th "Contributor Name"
+    th "Contributor Email"
+    th "Authority"
+  end
+  tr do
+    td a contributor.name, href:admin_contributor_path
```

Comment:
When I visit this link, what extra information do I get? This relates to my comment questioning if we need the new `app/admin/contributors.rb` admin section.